submissions/fabunmiTolulope/HTML/Productpage.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta name="keywords" content="banana, plantain, breeds of banana">
+	<meta name="author" content="Fabunmi Tolulope Eniola">
+	<title>Product page for Greenprint Integrated Services</title>
+</head>
+<body>
+<header>
+		<h1>Welcome to Greenprint Integrated Services</h1>
+</header>
+	<nav>
+		<ul>
+				<li><a href="#"> Home</a></li>
+				<li><a href="#">Blog</a> </li>
+				<li><a href="Productpage.html">Productpage</a> </li>
+				<li><a href="#">Contact Us</a> </li>
+		</ul>
+	</nav>
+	<hr>
+<section>
+	<h3> Organic Giant Cavendish Banana</h3>
+	<img src="cavendish2.jpg";>
+	<img src="cavendish.jpg";>
+	<p><strong> &#x20A6;10,000</strong></p>
+</section>
+<section>
+<p><h3> Product Description </h3>
+	From a cultivars that dominate the international trade in bananas, this <strong>Organic giant Cavendish Banana (<em>Musa sapenticum</em>)</strong> sets the standards in terms of taste and nutrient yield. <br>They are rich in Vitamin C, B6, minerals and dietary fibres. They are a rich energy source with 25%  carbohydrate. <br> It could be used for Banana Starch, flour and chips.<br> They can be eaten raw, used as smoothie and added to your homemade yummy yoghurt and Icecream.
+	<h3>Similar breeds of banana available</h3>
+	<ol>
+		<li type="i">Plantain (Cooking banana) </li>
+		<li type="i"> Grand Nain banana</li>
+		<li type="i"> Gros michel banana</li>
+		<li type="i"> Lady finger banana</li>
+		<li type="i"> <a href="#">  More items </a></li>
+	</ol> 
+</p><hr>
+</section>
+<nav>
+	<ul>
+		<li><a href="#"> Add to cart</a></li>
+		<li><a href="#">View more products</a> </li>
+	</ul>
+</nav>
+<FOOTER>
```

Comment:
This should be in lowercase. We want consistency

Reply:
i will effect that immediately

Reply:
lowercase now